src/resources/js/events-admin.js

```diff
@@ -25,14 +25,14 @@ var tribe_timepickers = tribe_timepickers || {};
 	obj.$ = {};
 
 	obj.container = function( k, container ) {
-		var $container  = $( container );
-		var $all_day    = $container.find( obj.selector.all_day );
-		var $timepicker = $container.find( obj.selector.timepicker );
-		var $timezone   = $container.find( obj.selector.timezone ).not( obj.selector.input );
-		var $input      = $container.find( obj.selector.timezone ).filter( obj.selector.input );
+		var $container = $( container ),
+			$all_day = $container.find( obj.selector.all_day ),
+			$timepicker = $container.find( obj.selector.timepicker ),
+			$timezone = $container.find( obj.selector.timezone ).not( obj.selector.input ),
+			$input = $container.find( obj.selector.timezone ).filter( obj.selector.input ),
 
-		// Create the Link
-		var $timezone_link = $( obj.timezone.link( { label: $input.data( 'timezoneLabel' ), timezone: $input.data( 'timezoneValue' ) } ) );
+			// Create the Link
+			$timezone_link = $( obj.timezone.link( { label: $input.data( 'timezoneLabel' ), timezone: $input.data( 'timezoneValue' ) } ) );
```

Comment:
As above, would be better as a standalone definition, ie: `var $timezone_link = $( whatever );`